src/components/my-page/IdeaMarketPostHeader.tsx

```diff
@@ -1,19 +1,30 @@
+import { useNavigate } from 'react-router-dom';
 import styles from './ideaMarketPostHeader.module.scss';
-import arrowButton from '../../assets/icons/arrow-button.svg';
 
 interface PostHeaderProps {
   category: string;
   title: string;
   price: number;
+  postId: number;
 }
 
 export const IdeaMarketPostHeader = ({
   category,
   title,
   price,
+  postId,
 }: PostHeaderProps) => {
+  const navigate = useNavigate();
   const FORMATTEDPRICE = price.toLocaleString();
 
+  const handleEditNavigate = () => {
+    if (postId) {
+      navigate(`/my/posts/idea-market/edit/${postId}`);
+    } else {
+      navigate('/my/posts//idea-market');
+    }
```

Comment:
요즘은 if else 보다 early return문을 지향하는 것 같아요!

```suggestion
    if (postId) {
      return navigate(`/my/posts/idea-market/edit/${postId}`);
    } 
      return navigate('/my/posts//idea-market');
  
```